index.html

```diff
@@ -2455,6 +2475,22 @@
             </li>
           </ol>
           <p>
+            When the <a>receiving browsing context</a> is closed, any
+            associated browsing state, including <a>session history</a>, the
+            <a>cookie store</a>, any <a>HTTP authentication</a> state, the
+            <a>application cache</a>, any <a>databases</a>,
+            <a><code>sessionStorage</code></a>, and
+            <a><code>localStorage</code></a> MUST be discarded and not used for
+            any other <a>receiving browsing context</a>.
+          </p>
+          <p class="note">
+            The <a>receiving browsing context</a> is not intended to define a
+            private browsing session, but a well defined environment to allow
+            interoperable behavior for 1-UA and 2-UA presentations. For
+            example, navigation is not allowed in a <a>receiving browsing
+            context</a>.
+          </p>
```

Comment:
In the note, instead of "The receiving browsing context", I would rather use "This algorithm" (or "These steps").

Also, readers won't have the full context of WG discussions, and "private browsing session" comes a bit out of the blue here. I would simply strike that part and simply assert the intent.

Where do we say that navigation is not allowed in a receiving browsing context? The auxiliary navigation browsing context flag merely prevents the creation of additional browsing contexts, for instance. What the spec rather says is that the receiving user agent must terminate the presentation in case of navigation.

All in all, I would simplify the note to:
```
<p class="note">  
  This algorithm is intended to create a well defined environment to allow interoperable behavior for 1-UA and 2-UA presentations.
</p>  
```